### ISSUE TITLE
help! 使用自定义数据集训练成功,但predict结果为全红的图片

### ISSUE THREAD

**Issue by Apokar** [PaddlePaddle/PaddleSeg#1213]
### **配置文件(根据bisenet_optic_disc_512x512_1k.yml修改):**

**_bisenet_new_camvid_512x512_1k.yml_**

> batch_size: 4
> iters: 1000
> 
> train_dataset:
>   type: Dataset
>   dataset_root: data/NewCamVid
>   train_path: data/NewCamVid/train.txt
>   num_classes: 2
>   transforms:
>     - type: Resize
>       target_size: [512, 512]
>     - type: RandomHorizontalFlip
>     - type: Normalize
>   mode: train
> 
> val_dataset:
>   type: Dataset
>   dataset_root: data/NewCamVid
>   val_path: data/NewCamVid/val.txt
>   num_classes: 2
>   transforms:
>     - type: Resize
>       target_size: [512, 512]
>     - type: Normalize
>   mode: val
> 
> optimizer:
>   type: sgd
>   momentum: 0.9
>   weight_decay: 4.0e-5
> 
> lr_scheduler:
>   type: PolynomialDecay
>   learning_rate: 0.01
>   end_lr: 0
>   power: 0.9
> 
> loss:
>   types:
>     - type: CrossEntropyLoss
>   coef: [1, 1, 1, 1, 1]
> 
> model:
>   type: BiSeNetV2
>   pretrained: Null

**训练的图片我贴上一对吧**

原数据
![0016E5_08139](https://user-images.githubusercontent.com/25199579/127449425-c923eb90-663b-472b-b521-db5806df0a9c.jpg)

标注数据
![0016E5_08139_L](https://user-images.githubusercontent.com/25199579/127449414-4911ec38-e03f-4abc-862b-15696753d2bc.png)

标注数据全都已转换为单通道灰度图  img.shape为(720,960)
天空部分的像素值为(1,1,1)
其余部分是(0,0,0)

使用训练出的模型运行 predict.py 后结果为:

![Seq05VD_f04740](https://user-images.githubusercontent.com/25199579/127449697-7424501c-82de-471d-8abf-b4c426e57985.jpg)

![Seq05VD_f04740](https://user-images.githubusercontent.com/25199579/127449703-83e93ca6-4d91-403a-b294-0060d37eaf54.png)

**请问下 这个是不是数据集有问题吗 还是其他地方不对呢?请指教!**

**Comment by Apokar** [comment]
发现一个错误 我去验证下 先关闭问题吧

**Comment by Apokar** [comment]
是我自己的问题哈哈  paddle还是一如既往的强悍啊

**Comment by Peng-ss** [comment]
> 是我自己的问题哈哈 paddle还是一如既往的强悍啊
你好，我也遇到同样的问题，你是如何解决的？
